api/src/main/java/com/epam/pipeline/manager/contextual/handler/DefaultContextualPreferenceReducer.java

```diff
@@ -57,15 +58,21 @@
     }
 
     private Optional<ContextualPreference> reduceValidPreferences(final List<ContextualPreference> preferences) {
-        return preferenceReducerMap.getOrDefault(preferences.get(0).getName(), fallbackEmptyReducer())
+        return preferenceReducerMap.getOrDefault(preferences.get(0).getName(), defaultReducer())
                 .reduce(preferences);
     }
 
-    private ContextualPreferenceReducer fallbackEmptyReducer() {
+    private ContextualPreferenceReducer defaultReducer() {
         return preferences -> {
-            log.warn(messageHelper.getMessage(MessageConstants.WARN_CONTEXTUAL_PREFERENCE_REDUCER_NOT_FOUND,
```

Comment:
We can get rid of `MessageConstants.WARN_CONTEXTUAL_PREFERENCE_REDUCER_NOT_FOUND` now.

Reply:
Thank you, fixed